rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/attachment/Sw360AttachmentService.java

```diff
@@ -128,6 +128,11 @@ public Attachment uploadAttachment(MultipartFile file, Attachment newAttachment,
             attachment.setCheckStatus(checkStatus);
         }
 
+		String createdComment = newAttachment.getCreatedComment();
+		if (createdComment != null) {
+			attachment.setCreatedComment(createdComment);
+		}
```

Comment:
Please remove white spaces and correct the indentation accordingly.

Reply:
Thanks @nutanv1contr  for the review comments . I have made the changes and committed the same.